hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java

```diff
@@ -141,20 +144,26 @@ private static OMClientRequest getOMAclRequest(OMRequest omRequest) {
         return new OMVolumeAddAclRequest(omRequest);
       } else if (ObjectType.BUCKET == type) {
         return new OMBucketAddAclRequest(omRequest);
+      } else if (type == ObjectType.KEY) {
```

Comment:
The code is using reverse order for equality. Can you use the same?

Reply:
Updated it.